newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -104,19 +119,30 @@ export default class AnimationPreview extends Component<Props, State> {
           resourcesLoader={resourcesLoader}
           project={project}
         />
-        <Line>
-          <Column expand>
-            <FlatButton
-              icon={<Replay />}
-              label="Replay"
-              onClick={this.replay}
-            />
-            <FlatButton
-              icon={paused ? <PlayArrow /> : <Pause />}
-              label={paused ? 'Play' : 'Pause'}
-              onClick={paused ? this.play : this.pause}
-            />
-          </Column>
+        <Line noMargin>
```

Comment:
Can you try `<Line noMargin alignItems="center">` (or `<Line noMargin alignItems="baseline">`) and removing the padding + bottom?

Reply:
@4ian cheers, that solves it! I didnt know about that parameter. No need for ugly css nudging code!